libs/data-mapper/src/lib/core/DataMapperDesignerProvider.tsx

```diff
@@ -49,34 +50,37 @@ export interface DataMapperDesignerProviderProps {
   children: React.ReactNode;
 }
 
-// NOTE: Leaving ThemeProvider here as we still use Fluent V8 components
 export const DataMapperDesignerProvider = ({ theme = 'light', locale = 'en', options, children }: DataMapperDesignerProviderProps) => {
   return (
     <AppInsightsContext.Provider value={reactPlugin}>
       <ReduxProvider store={store}>
         <DataMapperWrappedContext.Provider value={options}>
-          <FluentProvider
-            theme={theme === 'light' ? extendedWebLightTheme : extendedWebDarkTheme}
+          <ThemeProvider
+            theme={theme === 'light' ? AzureThemeLight : AzureThemeDark}
```

Comment:
Very small nit, maybe you can use the Theme enum in here `libs/utils/src/lib/models/theme.ts`

Reply:
@ccastrotrejo Could you take a look at why the tests are failing the designer-ui? The only changes I made that could've affected that were bumping the FluentUI versions, but I wouldn't think they'd do that much damage...